packages/path_provider/path_provider_windows/test/path_provider_windows_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path_provider_windows/path_provider_windows.dart';
+
+void main() {
+  PathProviderWindows pathProvider;
+
+  setUp(() {
+    pathProvider = PathProviderWindows();
+  });
+
+  tearDown(() {});
+
+  test('getTemporaryPath', () async {
+    expect(await pathProvider.getTemporaryPath(), contains('C:\\'));
+  });
```

Comment:
- We should have more tests here, so we're covering the API surface.
- You'll either need something like https://github.com/flutter/plugins/pull/2988/files#diff-f9349795ba45b74fe94e478e177c111aR15 around the Win32 calls themselves to allow stubbing them out, or change the tests to early-return on non-Windows platforms; these tests will be run on all platforms, and will fail at library lookup on macOS and Linux (you can see this on the last set of CI runs).